decred_wallet/Features/Wallet Setup/RecoverExistingWalletViewController.swift

```diff
@@ -111,55 +126,50 @@ class RecoverExistingWalletViewController: WalletSetupBaseViewController, UITabl
         if self.validateSeed().valid {
             self.activateConfirmButton()
         } else {
-            self.deactivateConfirmButton()
+            self.deactivateConfirmButtonForError()
         }
     }
     
     func focusSeedWordInput(at tableRowIndex: Int) {
         let tableIndexPath = IndexPath(row: tableRowIndex, section: 0)
-        
         let nextSeedWordCell = self.tableView.cellForRow(at: tableIndexPath) as? RecoveryWalletSeedWordCell
         nextSeedWordCell?.seedWordAutoComplete.becomeFirstResponder()
         
-        self.tableView.scrollToRow(at: tableIndexPath, at: .middle, animated: true)
+        self.tableView.scrollToRow(at: tableIndexPath, at: .bottom, animated: true)
```

Comment:
Is there a need to "always" move the user to the bottom? Android doesn't do this. I don't think the issue meant that the app should always scroll to the bottom but just like android, at some point the user will be taken to the bottom naturally.

Reply:
please check the issue created for it and it works accordingly